src/boot/ksdk1.1.0/warp-kl03-ksdk1.1-boot.c

```diff
@@ -850,6 +860,133 @@ setTPS82740CommonControlLines(uint16_t voltageMillivolts)
 }
 
 
+void
+debugISL23415(void)
+{
+#ifdef WARP_BUILD_ENABLE_DEVISL23415_DEBUG
+	for (uint8_t i=0; i < 4; i++) 
+	{
+		readDeviceRegisterISL23415(kWarpISL23415RegACR, 4);
+
+	#ifdef WARP_BUILD_ENABLE_SEGGER_RTT_PRINTF
+		SEGGER_RTT_printf(0, "\r\n\tRead %u ISL23415 ACR bytes: 0: 0x%02X, 1: 0x%02X, 2: 0x%02X, 3: 0x%02X, Status: %u", i,
+			deviceISL23415State.spiSinkBuffer[0], deviceISL23415State.spiSinkBuffer[1], deviceISL23415State.spiSinkBuffer[2], deviceISL23415State.spiSinkBuffer[3], deviceISL23415State.ksdk_spi_status);
+	#endif
+
+		readDeviceRegisterISL23415(kWarpISL23415RegWR, 4);
+
+	#ifdef WARP_BUILD_ENABLE_SEGGER_RTT_PRINTF
+		SEGGER_RTT_printf(0, "\r\n\tRead ISL23415 WR0 bytes: 0: 0x%02X, 1: 0x%02X, 2: 0x%02X, 3: 0x%02X, Status: %u\n", 
+			deviceISL23415State.spiSinkBuffer[0], deviceISL23415State.spiSinkBuffer[1], deviceISL23415State.spiSinkBuffer[2], deviceISL23415State.spiSinkBuffer[3], deviceISL23415State.ksdk_spi_status);
+	#endif
+	
+		OSA_TimeDelay(2000);
+	}
+
+	uint8_t valuesDCP[2] = {0x81, 0x79};

```

Comment:
Could you kindly add a comment about the intent for these values (even though this is a debug routine, that would be handy for others using it).

Reply:
They are random values, differing the least from the 0x80. I will place the comment, but I understand that this might not be the best choice of values.

Reply:
OK, kindly add a comment.